### ISSUE TITLE
Windows opening TumblrThree like a video?

### ISSUE THREAD

**Issue by Barbarbi7** [johanneszab/TumblThree#238]
Since I made the last windows update i have this weird bug with Tumblrthree:
![image](https://user-images.githubusercontent.com/39628072/40681085-6e17fcde-6388-11e8-8e10-338ed377d0dd.png)
Nothing happens when i click on this icon and windows seems to think that it's a video(play/pause/stop icon) :/

I have this issue only with Tumblrthree and:
- Downloading the last vession didn't change anything
- Run it as admin didn't change anything

Someone have an idea?

**Comment by Hrxn** [comment]
But otherwise everything works normally?

**Comment by Barbarbi7** [comment]
No, when I Open TumblrThree.exe, I only have the icon(as shown above).
When I click on the tumblrThree icon, tumblrThree window isn't shown. 
I can't use it.

**Comment by Hrxn** [comment]
Strange... do you have anything unusual running in the background?
Or some rather obscure applications? Alternatively, Virus protection software etc. can cause issues as well, although rather rare.

**Comment by johanneszab** [comment]
> windows seems to think that it's a video(play/pause/stop icon) :/

Those buttons have nothing to do with video. I've added them to TumblThree and they allow you to stop/resume/start/pause TumblThree.

As for resolving your issue, it has nothing to do with the recent Windows update or that TumblThree doesn't work in general. Thus, it must be something with your specific configuration. So, try to delete the settings, re-download TumblThree and basically start from scratch. If that works, re-apply your settings. If that works as well, re-add your blogs. You should find all the necessary information for doing this in the readme(front page)/on my website. Briefly, the settings are stored under `%localappdata%\TumblThree`, the blog relative information in `Your download location_\Index\`. You can simply move all those files while TumblThree is closed, and re-apply them by moving them back while TumblThree is closed.
